lib/core/src/lib/info-drawer/info-drawer.component.spec.ts

```diff
@@ -139,34 +140,44 @@ describe('Custom InfoDrawer', () => {
         fixture.detectChanges();
         const tab: any = fixture.debugElement.queryAll(By.css('.mat-tab-label-active'));
         expect(tab.length).toBe(1);
-        expect(tab[0].nativeElement.innerText).toContain('TAB1');
+        expect(tab[0].nativeElement.innerText).toContain('Tab1');
     });
 
     it('should select the tab 2 (index 1)', () => {
         component.tabIndex = 1;
         fixture.detectChanges();
         const tab: any = fixture.debugElement.queryAll(By.css('.mat-tab-label-active'));
         expect(tab.length).toBe(1);
-        expect(tab[0].nativeElement.innerText).toContain('TAB2');
+        expect(tab[0].nativeElement.innerText).toContain('Tab2');
     });
 
     it('should render a tab with icon', () => {
         component.tabIndex = 2;
         fixture.detectChanges();
         const tab: any = fixture.debugElement.queryAll(By.css('.mat-tab-label-active'));
-        expect(tab[0].nativeElement.innerText).not.toBe('TAB3');
+        expect(tab[0].nativeElement.innerText).not.toBe('Tab3');
         expect(tab[0].nativeElement.innerText).toContain('tab-icon');
     });
+
+    it('should render a icon with title', () => {
+        component.icon = '/assets/images/ft_ic_miscellaneous.svg';
+        fixture.detectChanges();
+        const icon =  fixture.debugElement.queryAll(By.css('[info-drawer-node-icon]'));
```

Comment:
this fixture.debugElement.queryAll(By.css('[info-drawer-node-icon]')); repeats couple times in that file, can we move it to function and reuse?

Reply:
Done